src/newlayout/components/Map/Controls/index.tsx

```diff
@@ -187,93 +188,98 @@ const MapControls = (props: IMapControlsProps) => {
   }, [poiFilter.selectedValues]);
 
   return (
-    <>
-      <Control position="topleft">
-        <Stack display={"flex"} direction={"column"} rowGap={1}>
-          <MapZoomControl />
-          <MapLayerControl showOnly={"mobile"} />
-          <HelpViewControl />
-        </Stack>
-      </Control>
-      <Control position="bottomleft">
-        <Stack display={"flex"} direction={"column"} rowGap={1}>
-          <MapTypeMapLayerViewComponent />
-          <MapLayerControl showOnly={"desktop"} />
-        </Stack>
-      </Control>
-      <Control position="topright">
-        <Stack
-          display={"flex"}
-          direction={"column"}
-          rowGap={2}
-          alignItems={"flex-end"}
-        >
-          <Stack display={"flex"} direction={"row"} columnGap={2}>
-            {/* <FilterButtonComponent
+    <DoubleClickStopPropagation>
```

Comment:
@oguzergul Isn't this the same as just passing a regular `onDoubleClick` handler to the div below? 🤔